test/native_test/native_test.dart

```diff
@@ -91,19 +91,20 @@ void main() {
     test('double', () {
       expect(bindings.Function1Double(0), 42.0);
     });
-    test('array-workaround: Order of access', () {
+    test('Array Test: Order of access', () {
       final struct1 = bindings.getStruct1();
       var expectedValue = 1;
-      for (var i = 0; i < struct1.ref.data.dimensions[0]; i++) {
```

Comment:
Ah the old api gave access to `.dimensions`? Then this is kind of a breaking change.

But we should likely add that kind of access to `Array` in `dart:ffi` as well.

Reply:
>  Then this is kind of a breaking change

Should I update the version to 3.0.0-beta then?

> But we should likely add that kind of access to Array in dart:ffi as well.

Yeah, I was wondering that there's no way to get the length of an array. (even though its constant and the user would know it beforehand)

Reply:
Hm, if we want to be true to semantic versioning, we probably should make it `3.0.0-beta.0`, good catch!

Reply:
Also, the key is probably rejected in the config right now, instead of just deprecated. So that is also a breaking change.

Reply:
> the key is probably rejected

It will simply warn about an unknown key called `array-workaound`, the tool would still work since an unknown key is not a severe error.

---

Updated version.

Reply:
I've also removed the previously deprecated `llvm-lib` config.